cli/package.json

```diff
@@ -27,6 +27,7 @@
         "maci-core": "^1.0.4",
         "maci-crypto": "^1.0.4",
         "maci-domainobjs": "^1.0.4",
+        "maci-common": "^1.0.0",
```

Comment:
nit: mind using `lerna` so all submodules share the same package version

Reply:
We probably won't publish this so not sure if it's necessarily required unless you imagine using it for other things in the `server` package

Reply:
some functions are used in both server and cli package, so I put the common functions in a separate folder. What I thought is to put the commonly used utility functions in this folder.